lib/internal/performance.js

```diff
@@ -0,0 +1,155 @@
+'use strict';
+
+const {
+  PerformanceMark,
+  PerformanceMeasure,
+  getEntries,
+  resetMark,
+  clearEntry,
+  nodeTiming,
+  nodeFrame,
+  nodeTimingKeys,
+  timeOrigin
+} = process.binding('performance');
+
+const kInspect = require('internal/util').customInspectSymbol;
+
+const marks = new Map();
+const measures = new Map();
+
+let errors;
+function lazyErrors() {
+  if (errors === undefined)
+    errors = require('internal/errors');
+  return errors;
+}
+
+class Performance {
+  constructor() {}
```

Comment:
No, it shouldn't. The constructor is not exposed to end users and those tho get there via `process.performance.constructor` won't hurt anything if they happen to create a new instance.